apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brev/detaljer/MottakerPanel.tsx

```diff
@@ -22,6 +27,15 @@ export default function MottakerPanel({ vedtaksbrev }: { vedtaksbrev: IBrev }) {
       )}
       <br />
 
+      {redigerbar && (
+        <RedigerMottakerModal
+          brev={brev}
+          oppdater={(val) => {
+            brev.mottaker = val
+            setBrev({ ...brev, mottaker: val })
```

Comment:
Er dette beste måten å gjøre det på? Har useState rundt brev slik at modalen blir oppdatert neste gang den åpnes.
Det beste hadde kanskje vært om RedigerMottakerModal tok inn en mottaker slik at man bare kunne gjøre setMottaker(mottaker) der setMottaker var en useState setter.

Reply:
Det er noe som skurrer litt, og jeg lurer på om det kan være noe problem knyttet opp til at brevet du tar inn som en prop ikke får oppdatert mottaker, og det er derfor du må sette manuelt. Men da ser jeg for meg at brevet som MottakerPanel tar imot uansett er utdatert og har en gammel mottaker, så hvis det brukes andre steder har det fremdeles gammel mottaker. Litt vanskelig å se om det er feil uten å se på bruken av MottakerPanel, men det er i alle fall potensiale for at vi holder på "feil" state for brevet en eller annen plass.

Reply:
Tenker å sende inn en callback-funksjon til MottakerPanel, så kan motttakerdata boble helt  tilbake til der man bruker MottakerPanel :) Kommer en commit på det